### PR TITLE
chore: Fix home viewAreaCoveragePercentThreshold fields and type

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

Fix home viewAreaCoveragePercentThreshold fields and type.

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Review by MounirDhahri — COMMENTED** [review]
The app seems to be breaking after your change. I am looking into patching rn to fix this because it's the expected behaviour
Screenshot 2023-04-06 at 18 39 57

File: src/app/Scenes/Home/Home.tsx [hunk 1]
```diff
@@ -131,10 +131,9 @@ const Home = memo((props: HomeProps) => {
 
   const { cards } = useContentCards()
 
-  const viewabilityConfig = useRef<FlatListProps<HomeModule>["viewabilityConfig"]>({
```
Comment (olerichter00) [inline src/app/Scenes/Home/Home.tsx:134]: `FlatListProps<HomeModule>["viewabilityConfig"]` as a type here doesn't seem to check for anything 🤔

File: src/app/Components/PrefetchFlatList.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function PrefetchFlatList<ItemType>({
 
   const viewabilityConfigRef = useRef({
     waitForInteraction: true,
-    viewAreaCoveragePercentThreshold: 0,
+    itemVisiblePercentThreshold: 0,
```
Comment (olerichter00) [inline src/app/Components/PrefetchFlatList.tsx:37]: Changing this to make sure we're not using both `viewAreaCoveragePercentThreshold` and `itemVisiblePercentThreshold` in one config.

**Review by MounirDhahri — APPROVED** [review]
Nice didn't know you're injecting it 😅

**Pull request merged**